docs/UserGuide.md

```diff
@@ -551,37 +569,38 @@ Acceptable values for `e/EMAIL`:
         - end with a domain label at least 2 characters long.
         - have each domain label start and end with alphanumeric characters.
         - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
-- Error message for invalid email: "Emails should contain two segments separated by an @ symbol."
+- Error message for invalid email: "Invalid command format!" followed by an instruction on the proper usage of the command you were executing.
 
 Acceptable values for `a/ADDRESS`:
 - Word with or without spaces.
+- Error message for invalid address: "Addresses can take any values, and it should not be blank".
 
 Acceptable values for `r/RSVP_STATUS`:
 - `yes`
 - `no`
 - `unknown`
 - Inputs with no values when adding a guest (e.g. `r/`) signify that RSVP status should be stored as `unknown`.
 - Inputs with no values when filtering guests (e.g. `r/`) signify that you wish to filter for guests with `unknown` RSVP status.
-- Error message for invalid RSVP status: “RSVP status can only be `yes`, `no` or `unknown`”.
+- Error message for invalid RSVP status: “RsvpStatus Status should only have one of three values. yes, no, or unknown.”.
 
 Acceptable values for `d/DIETARY_REQUIREMENT`:
 - Alphanumeric word with or without spaces.
 
 Acceptable values for `tn/TABLE_NUMBER`:
 - Non-negative integer with no spaces or special characters.
 - Preceding zeros will be trimmed.
-- Error message for invalid table numbers: "Table numbers should be positive numbers with no spaces or special characters."
+- Error message for invalid table numbers: "Table numbers should only contain numbers."
```

Comment:
Seems like the current error message has no full stop 😢 
```suggestion
- Error message for invalid table numbers: "Table numbers should only contain numbers"
```
<img width="655" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/442eb902-c3d2-4280-bc65-07ccfecdbbc1">

Reply:
Resolved!